index.html

```diff
@@ -15,10 +15,10 @@
 <body>
```

Comment:
- [ ] Great Job So far but There is something you still need to work on for your website to be accessible, I noticed that some of the text in your code has very low contrast with the background. This can make it difficult for users, especially those with visual impairments, to read the text. Can you please adjust the contrast to ensure that the text is legible and accessible for all users? wink if you wanna [check your contrast use this tool](https://www.aditus.io/button-contrast-checker/)
![Screenshot from 2023-03-01 00-16-14](https://user-images.githubusercontent.com/88894525/221994140-b8dc5b6d-cee2-4723-9b18-d35f54652825.png)